mpt_extension_sdk/pipeline/step.py

```
@@ -0,0 +1,48 @@
+        try:
+            await self.post(ctx)
+        except Exception as post_error:
+            if process_error is not None:
+                raise post_error from process_error
+            raise
+        if process_error is not None:
+            raise process_error
+
+    async def pre(self, ctx: ExecutionContext) -> None:
+        """Run pre-processing hook."""
+        return  # noqa: WPS324
```

Ruff (0.15.10)

[warning] Invalid rule code in `# noqa`: WPS324

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)